development/playbooks/setup-repositories/setup-repositories.yaml

```diff
@@ -9,10 +9,30 @@
         name: theforeman.operations.puppet_repositories
       vars:
         foreman_puppet_repositories_version: "8"
+      when:
+        - ansible_distribution_major_version == '9'
 
     - name: Setup Foreman repositories
       ansible.builtin.include_role:
         name: theforeman.operations.foreman_repositories
       vars:
         foreman_repositories_version: nightly
         foreman_repositories_katello_version: nightly
+      when:
+        - ansible_distribution_major_version == '9'
+
+    - name: Enable epel
+      ansible.builtin.package:
+        name: epel-release
+        state: installed
+      when:
+        - ansible_distribution_major_version == '10'
+
+    - name: Enable hammer-el10 copr
+      community.general.copr:
+        host: copr.fedorainfracloud.org
+        state: enabled
+        name: evgeni/hammer-el10
```

Comment:
you may know this copr from fixes like https://github.com/theforeman/foreman-packaging/pull/12499 and https://github.com/theforeman/foreman-packaging/pull/12500

Reply:
(this needs to be read in the voice of Troy McClure)